### ISSUE TITLE
Further 'NPM' and JS issues

### ISSUE THREAD

**Issue by stephenhoult** [YakTrack/YakTrack#221]
Thanks @AndrewFeeney 

`npm install` works as expected now, however I'm running into an issue when running `npm run dev`.

```
ERROR in ./resources/css/app.css (./node_modules/css-loader??ref--5-2!./node_modules/postcss-loader/src??postcss0!./resources/css/app.css)
Module build failed (from ./node_modules/postcss-loader/src/index.js):
SyntaxError

(15:17) The `rounded` class does not exist. If you're sure that `rounded` exists, make sure that any `@import` statements are being properly processed before Tailwind CSS sees your CSS, as `@apply` can only be used for classes in the same CSS tree.

  13 | 
  14 | .btn {
> 15 |     @apply px-4 rounded shadow no-underline btn-height;
     |                 ^
  16 | }
  17 | 

```

I can get around this by downgrading tailwind from 2.2.0 to 1.2.0 - but that's probably not the correct fix.

However this seems to be just the tip of the iceberg.

Once all the js / css is compiled and you're logged in there are issues with all navigation links.

E.g. when clicking on the Sessions icon in the sidebar.
![image](https://user-images.githubusercontent.com/33623309/137115336-d5455ff5-a834-4d3f-a6e8-9aa6fad3621c.png)

_Originally posted by @stephenhoult in https://github.com/YakTrack/YakTrack/issues/220#issuecomment-942107686_

**Comment by AndrewFeeney** [comment]
I appreciate your persistence with this! What a mess. I think I've addressed the dependency issues with #222 and will close this ticket accordingly. Those dependencies will need to be properly upgraded soon, but for now this should allow development to continue. Let me know if you run into any further issues with this.

**Comment by stephenhoult** [comment]
Thanks @AndrewFeeney  Will let you know if I experience any further issues.
